WordPress/src/main/res/layout/category_detail_fragment.xml

```diff
@@ -1,5 +1,73 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/category_detail"
     android:layout_width="match_parent"
-    android:layout_height="match_parent" />
+    android:layout_height="match_parent"
+    >
```

Comment:
🔍 

1. XML can be formatted.
2. Hardcoded `dimen` values `16dp` can be replaced with values from `dimens.xml`.
3. Explicit comments for the elements can be removed, they are self-explanatory from the element ids.

Reply:
👍  Done in 2b39bce

Reply:
hey, actually I meant formatting to remove blank lines and having closing `>` bracket at its's appropriate place. It seems now the elements are not arranged in the order they appear on the screen. Sorry for not conveying it explicitly in my comment.

Reply:
@ashiagr , No, no. No need to apologise. I did get that. I just pushed in a hurry. 😄 .  Have rectified it in aecb803